docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -241,8 +241,13 @@ import cocoapods.example.*
 1. Specify the name of a Pod library in the `pod()` function.
 
    In the configuration block, specify the cinterop options:
-    * `extraOpts` – to specify the list of options for a Pod library. For example, specific flags: `extraOpts = listOf("-compiler-option")`
-    * `packageName` – to specify the package name. If you specify this, you can import the library using the package name: `import <packageName>`.
+
+   * `extraOpts` – to specify the list of options for a Pod library, for example, a `-compiler-option` with specific flags.
+     
+     You can use `extraOpts = listOf("-compiler-option", "-fmodules")` here to consume Objective-C headers with `@import`
```

Comment:
Maybe highlight it somehow? And add a note that this feature is experimental.

Reply:
Thanks! I decided that the best way to do that is to create a separate section